src/main/java/cuchaz/enigma/gui/dialog/CrashDialog.java

```diff
@@ -33,7 +33,7 @@ private CrashDialog(JFrame parent) {
 		final Container pane = frame.getContentPane();
 		pane.setLayout(new BorderLayout());
 
-		JLabel label = new JLabel(Constants.NAME + " " + LangUtils.translate("crash.summary") + " =(");
+		JLabel label = new JLabel(String.format(LangUtils.translate("crash.summary"), new Object[] {Constants.NAME}));
```

Comment:
```suggestion
		JLabel label = new JLabel(String.format(LangUtils.translate("crash.summary"), Constants.NAME));
```
This is a vararg so you can just pass in an object than using explicit array creation

Reply:
Already fixed in another commit